### ISSUE TITLE
github-review-forge-pr-at-point from forge-topic-mode

### ISSUE THREAD

**Issue by agzam** [charignon/github-review#21]
`github-review-forge-pr-at-point` works from magit-status view, but when you're visiting a PR (when you press RET and it opens it in `forge-topic-mode`) it does not work.

**Comment by charignon** [comment]
Good point, feel free to send a PR to address this, I would be happy to review it.

**Comment by charignon** [comment]
We can use `(forge-current-topic)` to get the PR in that case and make it work. I will look into it.
